src/web-components/vf-stave.js

```diff
@@ -0,0 +1,177 @@
+// ## Description
+// 
+// This file implements `vf-stave`, the web component that resembles 
+// the `Stave` element. One `vf-stave` can have multiple `vf-voice` children. 
+// `vf-stave` is responsible for keeping track of its children and creating 
+// the voices once they finish generating their notes. 
+// Once the voices are created, `vf-stave` dispatches an event to its parent
+// `vf-system` to signal that it's ready to be created and added to the system. 
+
+import './vf-score';
+
+const template = document.createElement('template');
+template.innerHTML = `
+  <slot></slot>
```

Comment:
For such a simple shadow root, I'd just use innerHTML right after the call to `attachShadow`